service/awsconfig/v8/resource/cloudformation/resource_test.go

```diff
@@ -33,13 +42,45 @@ func Test_Resource_Cloudformation_GetCloudFormationTags(t *testing.T) {
 					Key:   aws.String("giantswarm.io/cluster"),
 					Value: aws.String("5xchu"),
 				},
+				{
+					Key:   aws.String("giantswarm.io/organization"),
+					Value: aws.String("giantswarm"),
+				},
+				{
+					Key:   aws.String("giantswarm.io/installation"),
+					Value: aws.String("test-install"),
+				},
 			},
 		},
 	}
 
 	for _, tc := range testCases {
 		t.Run(tc.description, func(t *testing.T) {
-			tags := getCloudFormationTags(tc.obj)
+			c := Config{}
```

Comment:
Maybe all the config setup until the installation name could be extracted out of the loop, it doesn't depend on `tc`, WDYT?

Reply:
Good point! Done.